file-manager/src/main/scala/it/pagopa/interop/commons/files/service/impl/S3ManagerImpl.scala

```diff
@@ -164,4 +168,10 @@ final class S3ManagerImpl(blockingExecutionContext: ExecutionContextExecutor)(
 
   def calcContentMd5(byteArray: Array[Byte]): String = new String(Base64.encodeBase64(DigestUtils.md5(byteArray)))
 
+  def createJWTPath(): String = {
```

Comment:
Consider [this](https://github.com/pagopa/interop-be-jobs/pull/153/files#r1321473500).

Reply:
> I think I should avoid this approach. I suggest two ways to prevent this `asInstanceOf`:
> 
>     * Put the definition of `createJWTPath` directly in the trait definition
> 
>     * Move the function under the `utils` module

I would prefer the first because makes sense that this function is located inside that module but putting the definition in the trait would require to implement it inside `FileManagerImpl` too; in this case how do you suggest to implement it there?

Or maybe would be simpler to create a `misc` object inside utils?

Reply:
IMO this function is not part of the file manager, it is just a feature built on top of it.
If possible I would suggest to create a `createJWTPath` which, using a generic `FileManager`, implements the functionality that you need

Reply:
> IMO this function is not part of the file manager, it is just a feature built on top of it. If possible I would suggest to create a `createJWTPath` which, using a generic `FileManager`, implements the functionality that you need

In this case then we can define it in `utils` as suggested by @beetlecrunch , it shouldn't need a generic FileManager since I'm not using any of its functions.